experimental/auth-logic/endorsement-release/input_policy_examples/github_actions_policy.auth_logic

```diff
@@ -0,0 +1,11 @@
+"GithubActionsPolicy" says {
+    "Provenance" canSay any_app
+        has_expected_hash_from(any_hash, "Provenance").
```

Comment:
Not sure I understand the intuitive meaning of `has_expected_hash_from` (i.e. why is the second argument always "Provenance"?); in general, how do we document such predicates?

Reply:
Here's an explanation: [link](https://github.com/project-oak/transparent-release/blob/0048299cf2cda06bd7b79eac0ed75a58ffb159d8/experimental/auth-logic/wrappers/verifier_wrapper.go#L34).

Authorzation logic is essentially being re-implemented in C++ as part of Raksha. This prototype is still using the Rust version. The new version of authorization logic for Raksha will also be iterated on, and it will fix this by allowing a restricted form of "says" on the RHS that should still be decidable. This is the workaround for now, but this can be fixed later once we migrate to the C++ version.

Reply:
If we really intend to do a blog post or some other external communication of this before migrating to the C++ version, I can make this change in the Rust version in time for that external comm, of course.